package.json

```diff
@@ -43,7 +43,8 @@
     "stats:imports": "node ./scripts/node/util/getImportStats.js",
     "type-check": "tsc",
     "semantic-release": "semantic-release",
-    "commit": "git-cz"
+    "commit": "git-cz",
+    "check": "eslint --max-warnings 0 ./src/ --ext .js,.jsx,.ts,.tsx && jest && tsc"
```

Comment:
[should] I would point to the existing yarn commands. There's a lint command, a test command, and a typecheck command. The single command should encapsulate all of these, that way if we change those, this single command will automatically be right. There should be some existing examples in this document.